packages/actor-rdf-metadata-aggregate-total-items/lib/ActorRdfMetadataAggregateTotalItems.ts

```diff
@@ -0,0 +1,69 @@
+import { ActorRdfMetadataAggregate, IActionRdfMetadataAggregate, IActorRdfMetadataAggregateOutput } from '@comunica/bus-rdf-metadata-aggregate';
+import { IActorArgs, IActorTest } from '@comunica/core';
+import {Record} from "immutable";
+
+/**
+ * A comunica Total Items RDF Metadata Aggregate Actor.
+ */
+export class ActorRdfMetadataAggregateTotalItems extends ActorRdfMetadataAggregate {
+  public records: Record<string,any>[] = [];
+
+  public constructor(args: IActorArgs<IActionRdfMetadataAggregate, IActorTest, IActorRdfMetadataAggregateOutput>) {
+    super(args);
+  }
+
+  public async test(action: IActionRdfMetadataAggregate): Promise<IActorTest> {
+    return true;
+  }
+
+  public async run(action: IActionRdfMetadataAggregate): Promise<IActorRdfMetadataAggregateOutput> {
+    const {metadata, subMetadata} = action
```

Comment:
Did you somehow disable the linter? Because I would expect the linter to complain about this line (and others).